CMakeLists.txt

```diff
@@ -221,6 +221,10 @@ if (WIN32 OR CLR_CMAKE_PLATFORM_LINUX)
   add_definitions(-DFEATURE_EVENT_TRACE=1)
 endif (WIN32 OR CLR_CMAKE_PLATFORM_LINUX)
 
+if (CLR_CMAKE_PLATFORM_LINUX)
+    add_definitions(-DFEATURE_EVENTSOURCE_XPLAT=1)
```

Comment:
You should name this to something relevant to Event tracing - FEATURE_EVENT_TRACE_XPLAT is an option - unless the second define is only for Eventsource.